graphistry/arrow_uploader.py

```diff
@@ -380,7 +380,17 @@ def verify(self, token=None) -> bool:
             json={'token': token})
         return out.status_code == requests.codes.ok
 
-    def create_dataset(self, json):  # noqa: F811
+    def validate_encoding(self, json):
+        edge_attribute = json['edge_encodings']['complex']['default']['edgeColorEncoding']['attribute']
```

Comment:
this would exn if the encoding is not defined, right?

Reply:
I think I fixed this by incorporating the server-side logic here, so done, thanks!